README.md

```diff
@@ -1,97 +1,46 @@
-1. Copy the [step template](https://github.com/bitrise-steplib/step-template) files into your repository
-1. Fill the `step.sh` with your functionality
-1. Wire out your inputs to `step.yml` (`inputs` section)
-1. Fill out the other parts of the `step.yml` too
-1. Provide test values for the inputs in the `bitrise.yml`
-1. Run your step with `bitrise run test` - if it works, you're ready
-
-**For Step development guidelines & best practices** check this documentation: [https://github.com/bitrise-io/bitrise/blob/master/\_docs/step-development-guideline.md](https://github.com/bitrise-io/bitrise/blob/master/_docs/step-development-guideline.md).
-
-**NOTE:**
-
-If you want to use your step in your project's `bitrise.yml`:
-
-1. git push the step into it's repository
-2. reference it in your `bitrise.yml` with the `git::PUBLIC-GIT-CLONE-URL@BRANCH` step reference style:
-
-```
-- git::https://github.com/user/my-step.git@branch:
-   title: My step
-   inputs:
-   - my_input_1: "my value 1"
-   - my_input_2: "my value 2"
-```
+## Configuration
 
-You can find more examples of step reference styles
-in the [bitrise CLI repository](https://github.com/bitrise-io/bitrise/blob/master/_examples/tutorials/steps-and-workflows/bitrise.yml#L65).
+### Inputs
 
-## How to contribute to this Step
+| Parameter | Description | Required | Default |
+| --- | --- | --- | --- |
+| exclude | Exclude patterns | - | "*/vendor/*" |
 
-1. Fork this repository
-1. `git clone` it
-1. Create a branch you'll work on
-1. To use/test the step just follow the **How to use this Step** section
-1. Do the changes you want to
-1. Run/test the step before sending your contribution
+### Outputs
 
-- You can also test the step in your `bitrise` project, either on your Mac or on [bitrise.io](https://www.bitrise.io)
-- You just have to replace the step ID in your project's `bitrise.yml` with either a relative path, or with a git URL format
-- (relative) path format: instead of `- original-step-id:` use `- path::./relative/path/of/script/on/your/Mac:`
-- direct git URL format: instead of `- original-step-id:` use `- git::https://github.com/user/step.git@branch:`
-- You can find more example of alternative step referencing at: https://github.com/bitrise-io/bitrise/blob/master/_examples/tutorials/steps-and-workflows/bitrise.yml
+| Environment Variable | Description |
+| --- | --- |
+| BITRISE_GO_PACKAGES | List of go packages |
 
-7. Once you're done just commit your changes & create a Pull Request
+## Contributing
 
-## Share your own Step
+We welcome [pull requests](/pulls) and [issues](/issues) against this repository. 
 
-You can share your Step or step version with the [bitrise CLI](https://github.com/bitrise-io/bitrise). If you use the `bitrise.yml` included in this repository, all you have to do is:
+For pull requests, work on your changes in a forked repository and use the bitrise cli to [run your tests locally](https://devcenter.bitrise.io/bitrise-cli/run-your-first-build/)
```

Comment:
This link explains how to use Bitrise CLI locally, but in the case of a local step development, an important bit is that you use a local file path ([`- path::./:`](https://devcenter.bitrise.io/bitrise-cli/steps/#the-path-source)) to refer to the local state of the step.

Reply:
This should also go into the devcenter guide I think